site/src/docs/getting-started/tutorials/migrate-to-4.0.0.mdx

```diff
@@ -195,75 +195,95 @@ There are updates to lineheights to improve readability especially on Headings.
 
+---
+
 ## Design kit
 
 This guide will help you understand how to transition an existing project from using an older version of HDS Figma Design Kit to using the latest 4.0.0 version.
 
 ### What’s changing?
-This HDS Design Kit update follows the changes deployed in the code for the 4.0.0 release of HDS. The major differences include significant changes to the CookieConsent, Dropdown and Tag components. The new Select and Multiselect components replace the old Dropdown.
 
-The line heights of headings have been updated in the HDS Design Kit to improve readability, especially with larger titles. We have improved color black-60 for better contrast by changing it from #666666 to #595959.
+This HDS Design Kit update follows the changes deployed in the code for the 4.0.0 release of HDS. The major differences include significant changes to the **CookieConsent**, **Dropdown** and **Tag** components. The new **Select** and **Multiselect** components replace the old Dropdown.
 
-The HDS Design Kit in Figma has undergone a major restructuring: now, each component is placed on its own page. This new organisation makes it much easier to find components without having to guess which group they belong to. Each component will also have better documentation, which may include main component, subcomponents, specifications, variant examples, and usage & prototype examples, depending on the need and documentation status. We hope these changes will better explain the features of each component and how they can be used.
+The line heights of **headings** have been updated in the HDS Design Kit to improve readability, especially with larger titles. We have improved color **black-60** for better contrast by changing it from #666666 to #595959.
 
-### Components that need replacing
-[Dropdown] The Dropdown component is now deprecated and will be removed in future versions. Replace it with Select or Multiselect.
+The HDS Design Kit in Figma has undergone **a major restructuring**: now, each component is placed on its own page. This new organisation makes it much easier to find components without having to guess which group they belong to. Each component will also have better documentation, which may include main component, subcomponents, specifications, variant examples, and usage & prototype examples, depending on the need and documentation status. We hope these changes will better explain the features of each component and how they can be used.
 
-[Cookie consent] The CookieConsent component has been replaced with a new version, and it needs to be manually updated.
+#### Components that need replacing
 
-Deprecated components needs replacing, follow Figma guide about swapping components.
+- **[Dropdown]** The Dropdown component is now deprecated and will be removed in future versions. Replace it with Select or Multiselect.
```

Comment:
I would stylize the text when talking about components the same way that in the Dev part of this guide, so without the [ ]. Reading the whole guide I got a bit confused about this.

 I think we should also standardize when to use `code bit` because usually it's reserved for variables/tokens, but on this page it's also indicating subcomponents. Should it be used for main components too?